python/cuml/cuml/accel/tests/scikit-learn/xfail-list.yaml

```diff
@@ -1482,3 +1493,76 @@
+  - "sklearn.tests.test_common::test_estimators[SVC()-check_classifier_data_not_an_array]"
+  - "sklearn.tests.test_common::test_estimators[SVC()-check_classifiers_one_label]"
+  - "sklearn.tests.test_common::test_estimators[SVC()-check_classifiers_classes]"
+  - "sklearn.tests.test_common::test_estimators[SVC()-check_classifiers_train]"
+  - "sklearn.tests.test_common::test_estimators[SVC()-check_classifiers_train(readonly_memmap=True)]"
+  - "sklearn.tests.test_common::test_estimators[SVC()-check_classifiers_train(readonly_memmap=True,X_dtype=float32)]"
+  - "sklearn.tests.test_common::test_estimators[SVC()-check_supervised_y_no_nan]"
+  - "sklearn.tests.test_common::test_estimators[SVC()-check_decision_proba_consistency]"
+  - "sklearn.tests.test_common::test_estimators[SVC()-check_methods_sample_order_invariance]"
+  - "sklearn.tests.test_common::test_estimators[SVC()-check_methods_subset_invariance]"
+  - "sklearn.tests.test_common::test_estimators[SVC()-check_fit2d_1sample]"
+  - "sklearn.tests.test_common::test_estimators[SVC()-check_dict_unchanged]"
+  - "sklearn.tests.test_common::test_estimators[SVC()-check_fit_idempotent]"
+  - "sklearn.tests.test_common::test_estimators[SVC()-check_fit2d_predict1d]"
+  - "sklearn.tests.test_common::test_estimators[SVC()-check_requires_y_none]"
+  - "sklearn.svm.tests.test_svm::test_bad_input[lil_matrix]"
+  - "sklearn.svm.tests.test_svm::test_bad_input[lil_array]"
+  - "sklearn.svm.tests.test_svm::test_svc_nonfinite_params"
+  - "sklearn.svm.tests.test_svm::test_custom_kernel_not_array_input[SVC]"
+- reason: "SVM missing certain fit attributes"
+  marker: cuml_accel_svm_missing_fit_attributes
+  tests:
+  - "sklearn.svm.tests.test_svm::test_n_iter_libsvm[dataset0-SVC-ndarray]"
+  - "sklearn.svm.tests.test_svm::test_n_iter_libsvm[dataset0-SVR-int]"
+  - "sklearn.svm.tests.test_svm::test_n_iter_libsvm[dataset1-SVR-int]"
+  - "sklearn.svm.tests.test_svm::test_n_iter_libsvm[dataset2-SVR-int]"
+- reason: "SVM doesn't handle sample_weight identically to sklearn"
+  marker: cuml_accel_svm_sample_weight
+  tests:
+  - "sklearn.svm.tests.test_svm::test_weight"
+  - "sklearn.svm.tests.test_svm::test_negative_sample_weights_mask_all_samples[weights-are-zero-SVC]"
+  - "sklearn.svm.tests.test_svm::test_negative_sample_weights_mask_all_samples[weights-are-zero-SVR]"
+  - "sklearn.svm.tests.test_svm::test_negative_sample_weights_mask_all_samples[weights-are-negative-SVC]"
+  - "sklearn.svm.tests.test_svm::test_negative_sample_weights_mask_all_samples[weights-are-negative-SVR]"
+  - "sklearn.svm.tests.test_svm::test_negative_weights_svc_leave_just_one_label[mask-label-1-SVC]"
+  - "sklearn.svm.tests.test_svm::test_negative_weights_svc_leave_just_one_label[mask-label-2-SVC]"
+- reason: "Haven't implemented support for methods that conditionally exist"
+  marker: cuml_accel_conditional_methods
+  tests:
+  - "sklearn.svm.tests.test_svm::test_hasattr_predict_proba"
```

Comment:
From the sklearn test failures it looks like I'll have to resolve this one before merge :/.